Dockerfile

```diff
@@ -6,70 +6,71 @@
 # Pull base image.
 FROM phusion/baseimage:0.9.22
 
-# Set correct environment variables.
+LABEL name="Check If You Can Get Legal Aid (cla_public)" \
+      maintainer="LAA Get Access <laa-get-access@digital.justice.gov.uk>" \
+      version="1.0"
+
 ENV HOME /root
```

Comment:
Is this used anywhere?

Reply:
I think I would categorise this on the same level as `APP_HOME`. This might be a convention enforced by the base image and as the intent of this pull request is to optimise the Dockerfile, I glanced over it :) Could we defer it to another pull, please?

Reply:
👍